src/compass/s1_resample.py

```diff
@@ -11,6 +11,7 @@
 from osgeo import gdal
 
 from compass.utils.helpers import get_module_name
+from compass.utils.lut import compute_lut
```

Comment:
From my understanding, I'm not sure resample is the right place for the LUT2d correction to occur in the radar workflow. Based on the `isce3::geocode::geocodeSlc` [implementation](https://github-fn.jpl.nasa.gov/isce-3/isce/pull/1089). I think the corresponding location for these corrections should eventually be in [`s1_geocode`](https://github.com/opera-adt/COMPASS/pull/48/files). I'm assuming the radar workflow will eventually be geocoded?

Another issue with putting the corrections in resample, is how will these corrections be applied to the reference datasets, since only secondary datasets get resampled?

Reply:
Good point. One solution would be to remove the corrections from the radar workflow at the moment. You are correct, at a certain point we would want to geocode the burst on a geographical grid. However, for doing that, we will likely be using the `isce3.geocode.geocode_CFloat32()`. The core module does not support the use of correction LUTs at the moment. One solution would be to use `isce3.geocode.geocodeSlc()` and turn off the azimuth carrier resampling. Anyhow, I will remove the LUT computation from the radar workflow until we make a decision